app/assets/stylesheets/_layout.scss

```diff
@@ -0,0 +1,19 @@
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+figure {
+  margin: 0;
+}
+
+img,
+picture {
```

Comment:
Right that's why I said split this off from the styles above. I agree that `box-sizing` is layout but I don't think the `margin` and `max-width` are here. `layout` is not where I would look for `img` and `picture` styles.